composer.json

```diff
@@ -29,7 +29,7 @@
         "doctrine/data-fixtures": "<1.3"
     },
     "require-dev": {
-        "doctrine/coding-standard": "^6.0",
+        "doctrine/coding-standard": "^8.0",
```

Comment:
I let https://github.com/doctrine/DoctrineMongoDBBundle/pull/666 fix all coding style

Reply:
#666 was merged, I think there is no need to modify those tests anymore.